### ISSUE TITLE
fixing positioning and UI

### ISSUE THREAD

**Issue by R0Y15** [astush1001/Calculate-Me#8]
I'd like to contribute here, please assign this to me.

**Comment by R0Y15** [comment]
I'd like to contribute here, please assign this to me.

**Comment by R0Y15** [comment]
please label this as hacktoberfest and resolve the issue after labelling @astush1001

**Comment by mohitsaini07** [comment]
Hi @astush1001, Is this still available ?

**Comment by astush1001** [comment]
Yeah @mohitsaini07 I want some effects or some other elements to make it an attractive UI.
See if you can add any.

**Comment by vanshajkataria** [comment]
Hey can I try to improve colors and page design under this issue?

**Comment by vanshajkataria** [comment]
Also can I use Tailwind with this?

**Comment by astush1001** [comment]
Okay @vanshajkataria .. you can also work on this issue.

**Comment by mohitsaini07** [comment]
Thanks @astush1001
